src/Resources/config/services.yaml

```diff
@@ -6,13 +6,21 @@ services:
 
     CodeRhapsodie\EzDataflowBundle\Controller\DashboardController:
         public: true
+        autoconfigure: true
+        autowire: true
```

Comment:
Services from reusable bundles should not use autowiring or autoconfiguration.

https://symfony.com/doc/current/bundles/best_practices.html#services

Reply:
changed